Targets/AppStoreConnectUI/Sources/Authorization/AuthViewModel.swift

```diff
@@ -7,26 +7,34 @@
 
 import Foundation
 
-final class AuthViewModel: ObservableObject {
+public final class AuthViewModel: ObservableObject {
     private let authorization: Authorization
     
     @Published var issuerID = ""
-    @Published var privateKeyID = ""
-    @Published var privateKey = ""
+    @Published var privateKeyFileURL: URL? = nil
     
-    init(authorization: Authorization) {
+    public init(authorization: Authorization) {
         self.authorization = authorization
     }
     
-    func authorize() async throws {
-        try await authorization.authorize(issuerID: issuerID,
-                                          privateKeyID: privateKeyID,
-                                          privateKey: privateKey)
+    func authorize() async {
+        guard let privateKeyFileURL,
+              !issuerID.isEmpty
+        else {
+            assertionFailure("privateKeyFileURL is not set or issuerID is empty")
+            return
+        }
+
+        do {
+            try await authorization.authorize(issuerID: issuerID,
+                                              privateKeyFileURL: privateKeyFileURL)
+        }
+        catch {
+            print("\(error)")
```

Comment:
Let's add into US state this alert? Right now it could be system alert above the window, so you could store simple error

```suggestion
authorizationAllert  = error
            print("\(error)")
```